frontend/src/static/js/components/webstatus-stats-missing-one-impl-chart-panel.ts

```diff
@@ -159,8 +174,10 @@ export class WebstatusStatsMissingOneImplChartPanel extends WebstatusLineChartPa
   pointSelectedTaskRenderOnSuccess(): TemplateResult {
     return html`
       <div slot="header" id="${this.getPanelID()}-list-header">
-        The missing feature IDs on ${this.selectedDate} for
-        ${this.selectedBrowser}:
+        <a href="${this.featureListHref}">
```

Comment:
Can you add `target=”_blank”`?

Reply:
An edge case to consider: In the event a given browser actually burns down the list to zero, there will still be data points for that browser. The current anchor tag will have a href = `""` which will reload the page. We should handle the `!featureList.length` case  better by either removing the <a> tag and/or showing a different message completely (e.g. No features for this.selectedDate)

Reply:
Thanks! Now it will show `No missing features for on 2024-01-01 for chrome` instead when feature list is empty

Reply:
> Can you add `target=”_blank”`?

What is the argument for this? Do we open any other in-app link in a separate tab?

Reply:
Most of our in-app links are navigated within the tab. We do have links to a separate tab when it is an external site, e.g. GitHub, or about pages. It makes more sense to open a separate tab because users would lose the point selected (the tooltip) on the graph if it is navigated within a tab, e.g. click a point on the graph, navigate to a new page (overview or feature detail page) within a tab and go back.

Reply:
I am not sure this is sufficient justification for the difference in behavior. I don't see any compelling use cases for keeping the selected position in the chart while browsing the feature list. Once someone is done with that new tab, they will presumably click on another point (date or browser), which makes the existing chart state irrelevant.

I guess this is a question where we should get input from product management.

Reply:
SG. I will bring this up in our Eng meeting on Monday first!